jest.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   coverageThreshold: {
     global: {
       statements: 99.02,
-      branches: 97.90,
+      branches: 97.89,
```

Comment:
Why would we want to lower the thresholds?

Reply:
Removing a (previously tested) code branch decreased coverage. 

Because I want to keep the PR clean with only changes to relevant code, I decreased the coverage threshold.